docs/user-documentation/paged-content.md

```diff
@@ -0,0 +1,73 @@
+Paged content, books, periodicals, photographs with the front and back, etcetera, can use the members structure.
+(See [the section describing resource node membership](resource-nodes.md#members).)
+This involves creating a resource node for the root record (e.g. the book or the photograph)
+and child resource nodes for each sub-component (e.g. "Page 1" and "Page 2" or "recto" and "verso")
+with their corresponding media. Each "child" resource node contains a reference to their "parent" resource node
+using the `field_member_of` property. 
+
+!!! Note "Child v. Member"
+    Islandora 8 uses the "child" and "member" descriptor for resource nodes that 
+    store a reference to another resource node in the "Member Of" field interchangeably. 
+    Administrators will more often see the "member" terminology more often while 
+    front-end users will usually see "child" terminology.
+
+Similar to the collection view showing members of a collection, Islandora provides
+an Islandora Model terms, _Paged Content_ and _Page_, that can be used to trigger paged
+content behaviors. Islandora Defaults provides a Context for _Paged Content_ 
+resource nodes to trigger displaying an OpenSeadragon viewer showing the child 
+resource nodes' service files.
+
```

Comment:
Hi Seth - I think this paragraph helps a lot. I was able to load paged content and use the viewer to verify that the files are all visible, etc. Thanks.